src/test/java/bio/terra/service/dataset/DatasetIntegrationTest.java

```diff
@@ -468,12 +468,70 @@ public void testCombinedMetadataDataIngest() throws Exception {
             .ignoreUnknownValues(false)
             .maxBadRecords(0)
             .table("sample_vcf")
-            .path("gs://jade-testdata-useastregion/dataset-ingest-combined-control.json");
+            .path(
+                "gs://jade-testdata-useastregion/dataset-ingest-combined-control-duplicates.json");
```

Comment:
I wonder if this is a good practice for this test. The test needs to make sure that the json is in a readable bucket for the test to run, and that the json contains a certain set of data. Could this test copy the json to the cloud (using a random filename) as part of the test run (and then delete it once it's done using it)? That would make the test independent of the bucket contents and allow us to change the json without disrupting the test on other branches.

Reply:
https://broadworkbench.atlassian.net/browse/DR-2159